README.md

```diff
@@ -7,51 +7,56 @@ Originally developed as [match4healthcare](https://github.com/match4everyone/mat
+- Load test data (file needs to exist inside backend folder):
+`docker-compose exec backend django-admin loaddata fixture.json`
 
 File changes in python files trigger an auto-reload of the server.
-Migrations have to be executed with `docker exec backend python3 manage.py migrate`.
+Migrations are automatically executed when the container starts.
 
-After changes to the Docker configuration, you have to restart and build the containers with `docker-compose -f docker-compose.dev.yml up --build`.
+After changes to the Docker configuration, you have to restart and build the containers with `docker-compose up --build`.
 
 ### Production
-Set `SECRET_KEY`, `SENDGRID_API_KEY` and `MAPBOX_TOKEN`in `backend.prod.env` for Django
-`POSTGRES_DB`, `POSTGRES_USER`, `POSTGRES_PASSWORD`  inside `database.prod.env` for postgres on your host machine.
-Also add a `SLACK_LOG_WEBHOOK` to enable slack logging.
 
-To run a container in production and in a new environment execute the `deploy.sh` script which builds the containers, runs all configurations and starts the web service.
+## Reverse Proxy
+
+We recommend running the gunicorn server behind a reverse proxy to provide ssl and possibly run multiple services on one server.
+The default configuration will make the docker container reachable on port 8000 only on 127.0.0.1.
+
+A sample nginx configuration can be found at ./tools/nginx-sample-site.
+
+## Setup
+Copy `backend.prod.env.example` to `backend.prod.env` and set variables as documented in the example file for Django
+Copy `database.prod.env.example` to `database.prod.env` and set variables as documented in the example file for postgres on your host machine.
+
+To run a container in production and in a new environment execute the `docker-compose -f docker-compose.yml -f docker-compose.prod.yml up -d --build` script which builds the containers, runs all configurations and starts the web service.
 
 If you want to deploy manually follow these steps closly:
 
-1. Build the containers
-(Run `export CURRENT_UID=$(id -u):$(id -g)` if you want to run the backend as non-root)
-`docker-compose -f docker-compose.dev.yml -f docker-compose.prod.yml up -d --build`
-2. Make messages
-`docker exec --env PYTHONPATH="/match4everyone-backend:$PYTHONPATH" backend django-admin makemessages --no-location`
-3. Compile messages
-`docker exec --env PYTHONPATH="/match4everyone-backend:$PYTHONPATH" backend django-admin compilemessages`
-4. Collect static
-`docker exec backend python3 manage.py collectstatic --no-input`
-5. Migrate
-`docker exec backend python3 manage.py migrate`
-5. Check if all the variables are correct
-`docker exec backend python3 manage.py check`
-6. Restart the backend container (important, whitenoise does not reload static files after it has started)
-`docker-compose -f docker-compose.dev.yml -f docker-compose.prod.yml down && docker-compose -f docker-compose.dev.yml -f docker-compose.prod.yml up -d`
+#### Build the containers
```

Comment:
This section does not really describe how to build it manually as the docker image runs the steps automatically. Maybe we should replace the `docker-compose` command with something without docker and rename the section to "Setup without docker" or something

Reply:
@maltezacharias let's just leave it for now. The added information is good and the restructuring of the README should be part of #33 as it is a general issue.